dts/arm/nxp/nxp_mcxw71.dtsi

```diff
@@ -24,17 +24,11 @@
 
 &stcm {
 	ranges = <0x0 0x30000000 DT_SIZE_K(112)>;
-
-	stcm1: system_memory@1a000 {
-		compatible = "zephyr,memory-region", "mmio-sram";
-		reg = <0x1a000 DT_SIZE_K(8)>;
-		zephyr,memory-region = "RetainedMem";
```

Comment:
These 8KB is from independent power switch domain which can keep data retention in all power modes. So it should be left for low-power use cases, if any. I suggest creating a board overlay.
I faced the same issue here: https://github.com/zephyrproject-rtos/zephyr/pull/83195#discussion_r1941153619